### DISCUSSION TITLE
Multi-modal solution process - center on base?

### DISCUSSION THREAD

**Discussion by cnicol-gwlogic** [usgs/pestpp#253] [Q&A] [answered]
Hi,

If using the Multi-modal solution process, should I turn off the "center on base" option? Are there any potential issues with leaving this on? I'm thinking that for upgrade realisation groups that do happen to contain the base realisation, I'd still like to center on base; but will this mess with upgrade groups in which the base realisation is not a member, or does ies multi-modal cater for this situation naturally?

Thanks

**Comment by jtwhite79** [comment, marked as answer]
Hey @cnicol-gwlogic - great question.  The multi-modal solution effectively swaps out each realization as the "center" so that derivative approximation is most valid for that realization.  So IES should just ignore the "center on base" for the multi-modal solution...

**Reply by cnicol-gwlogic** [reply]
Champ - thanks a lot for the quick and clear reply, as always.
